client/src/utils/tone/index.ts

```diff
@@ -1,4 +1,5 @@
 import store from 'store';
+import * as Tone from 'tone';
```

Comment:
```suggestion
```
What's the rationale for this change?  We deliberately do not statically import `tone` because it's a large library that not all users need to view the site.

Reply:
I used this way of import b'cuz it is mentioned in npm.
I did some research and found a better way to  import  is `import { loaded, Player } from 'tone';`.  
Taking advantage of [tree shaking](https://webpack.js.org/guides/tree-shaking/).
If it seem fit I can change it or revert previous way importing.